sdk/storage/azure-storage-files-shares/vcpkg/vcpkg.json

```diff
@@ -18,7 +18,7 @@
     {
       "name": "azure-storage-common-cpp",
       "default-features": false,
-      "version>=": "12.8.0"
+      "version>=": "12.9.0"
```

Comment:
Is only files-shares (+ common) shipping?
Do we need to bump up the dependency on storage-common in the other storage blobs, datalake, queues packages?

Reply:
Yeah, in our previous release process, we wouldn't release a new version just because of an upgrade of dependency unless the module has new feature support or the common package has significant new features.